concrete_sigar_test.go

```diff
@@ -74,3 +74,18 @@ func TestConcreteFileSystemUsage(t *testing.T) {
 	fsusage, err = concreteSigar.GetFileSystemUsage("T O T A L L Y B O G U S")
 	assert.Error(t, err)
 }
+
+func TestConcreteGetFDUsage(t *testing.T) {
+	concreteSigar := &sigar.ConcreteSigar{}
+	fdUsage, err := concreteSigar.GetFDUsage()
+	// if it's not implemented, don't test
+	if _, ok := err.(*sigar.ErrNotImplemented); ok {
+		println("Skipping *ConcreteSigar.GetFDUsage test because it is no implemented for " + runtime.GOOS)
```

Comment:
Instead of `println`, you can you [`testing.Skipf()`](https://golang.org/pkg/testing/#B.Skipf).